Content.Server/Mousetrap/MousetrapComponent.cs

```diff
@@ -0,0 +1,25 @@
+using Content.Shared.Damage;
+using Content.Shared.Sound;
+
+namespace Content.Server.Mousetrap;
+
+[RegisterComponent]
+public sealed class MousetrapComponent : Component
+{
+    [ViewVariables]
+    public bool IsActive;
+
+    [DataField("soundOnActivate")]
+    public SoundSpecifier SoundOnActivate = new SoundPathSpecifier("/Audio/Items/snap.ogg");
```

Comment:
this should really be moved out of `MousetrapComponent` and just use the generic `EmitSoundOnTrigger` that already exists